### ISSUE TITLE
where is /hoot-server?

### ISSUE THREAD

**Issue by d3netxer** [ngageoint/hootenanny-ui#1862]
via the Readme: 

"You can start a preconfigured server by doing the following:

cd into /hoot-server"

**Comment by d3netxer** [comment]
oh, it's here: https://github.com/ngageoint/hootenanny-ui/tree/9b336f46b9a2f025639b8b2cdd5831da93131bf8

**Comment by d3netxer** [comment]
Actually, I would like to run this on a more recent branch, can there be a hoot-server directory in the hoot2x branch?

**Comment by bobi7** [comment]
We are no longer using that old hoot-server directory way. hoot2x is the main branch for the UI and you can read the README file here: https://github.com/ngageoint/hootenanny-ui/blob/hoot2x/README.md

the current UI is using webpack and npm so you can start up the ui with `npm start`.

I will eventually work on a docker container to contain all the various parts of the project all in 1 container but until then you will need to go to the [hootenanny](https://github.com/ngageoint/hootenanny) repo and setup hootenanny (both the java services and the c++ core) through either [vagrant](https://github.com/ngageoint/hootenanny/blob/master/docs/user/VAGRANT.md) or [RPM](https://github.com/ngageoint/hootenanny/blob/master/docs/install/HootenannyInstall.asciidoc)

There is this [documentation](https://github.com/ngageoint/hootenanny/blob/master/README.md#documentation) that can help guide you

**Comment by d3netxer** [comment]
Nice, I can't wait for the docker container to be released. 

I tried to setup hootenanny through vagrant, first locally. I have Windows, and I had issues with the symbolic links. I enabled symbolic links on my virtual machine and it got through the initial errors, but in the end it was not successful as I got an error similar to what @brianhatchl got at the end of this issue: https://github.com/ngageoint/hootenanny/issues/3527

However, it is not a big deal if I cannot get it to work locally, I can move on to trying to use vagrant to set it up on AWS. I still need to do some research on how to do this, in the meantime if you have any examples of how to set the correct environment variables that would be appreciated.
